src/test/suite/_completion/completion.tasks.test.ts

```diff
@@ -31,7 +31,7 @@ suite('Should do completion in tasks.json', () => {
 
 	var testTraits = test('Completes for traits', async () => {
 		assumeNotOnJenkins(testTraits);
-		const expectedCompletion = { label: 'platform', documentation: `The platform trait is a base trait that is used to assign an integration platform to an integration.In case the platform is missing, the trait is allowed to create a default platform.This feature is especially useful in contexts where there's no need to provide a custom configuration for the platform(e.g. on OpenShift the default settings work, since there's an embedded container image registry).` };
+		const expectedCompletion = { label: 'platform', documentation: `The platform trait is a base trait that is used to assign an integration platform to an integration. In case the platform is missing, the trait is allowed to create a default platform. This feature is especially useful in contexts where there's no need to provide a custom configuration for the platform (e.g. on OpenShift the default settings work, since there's an embedded container image registry).` };
```

Comment:
The reason for the broken test was just a change in the documentation (basically some spaces which have been inserted).